modules/hello-world/pages/overview.adoc

```diff
@@ -3,19 +3,10 @@
 :page-role: tiles
 :!sectids:
 
-= Go Analytics SDK
-
-
-
-
-
 The Analytics Go SDK allows you to connect to a xref:analytics:intro:intro.adoc[Enterprise Analytics] cluster from Go.
 For connecting to a Couchbase Server Cluster -- self-managed, or Capella Operational --
 see our xref:go-sdk:hello-world:overview.adoc[Go Operational SDK].
 
-TIP: Connecting to an xref:columnar:intro:intro.adoc[_Capella_ Analytics] cluster from Node.js currently requires a xref:nodejs-columnar-sdk:hello-world:overview.adoc[Node.js Columnar SDK].
```

Comment:
Is there a reason for removing this?
Capella Analytics will still need a Columnar SDK for the next few months,
so we should mention it somewhere.  :thinking:

Reply:
It mentioned node and I just didn't know if it was because there is no node analytics SDK.